src/components/rating-group-advanced.tsx

```
@@ -0,0 +1,190 @@
+                disabled={disabled || readOnly}
+              >
+                {allowHalf ? (
+                  <div className="relative">
+                    <div
+                      className="absolute inset-0 w-1/2 z-10 cursor-pointer"
+                      onMouseEnter={() => handleMouseEnter(index, true)}
+                      onClick={(e) => {
+                        e.stopPropagation();
+                        handleClick(index, true);
+                      }}
+                    />
```

Biome (2.4.4)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)